proto_utils.h uses deprecated ByteSize API

When compiling grpc against a new version of protobuf (3.11.2), gcc complains that `ByteSize` is deprecated because it is used in `proto_utils.h`.

```
proto_utils.h:52:31: error: ‘int google::protobuf::MessageLite::ByteSize() const’ is deprecated: Please use ByteSizeLong() instead [-Werror=deprecated-declarations]
```

Currently this can only be remedied by disabling the warning with `-Wno-deprecated-declarations`.

--- a/include/grpcpp/impl/codegen/proto_utils.h
+++ b/include/grpcpp/impl/codegen/proto_utils.h
@@ -49,7 +49,7 @@ Status GenericSerialize(const grpc::protobuf::MessageLite& msg, ByteBuffer* bb,
                 "ProtoBufferWriter must be a subclass of "
                 "::protobuf::io::ZeroCopyOutputStream");
   *own_buffer = true;
-  int byte_size = msg.ByteSize();
+  int byte_size = msg.ByteSizeLong();
   if ((size_t)byte_size <= GRPC_SLICE_INLINED_SIZE) {
     Slice slice(byte_size);
     // We serialize directly into the allocated slices memory

